supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1024,58 +661,36 @@
 
 *Use it*: yes
 
-*Incorrect forms*: Any form omitting the spaces, hyphen or correct capitalization
+*Incorrect forms*:
 
-*See also*: xref:elytron[elytron], xref:elytron-heading[Elytron]
+*See also*: xref:elytron[elytron]
 
 [discrete]
 [[security-manager]]
-==== security-manager (noun)
-*Description*: The "security-manager" subsystem is used to configure security policies used by the Java Security Manager. Write in lowercase as two words separated by a hyphen. See xref:security-manager-heading[Security Manager] entry for the correct usage in headings.
-
-*Use it*: yes
-
-*Incorrect forms*: Any form using uppercase letters or omitting the hyphen
-
-*See also*: xref:security-manager-heading[Security Manager]
-
-[discrete]
-[[security-manager-heading]]
-==== Security Manager (noun)
-*Description*: Use "Security Manager" when referring to the xref:security-manager[security-manager] subsystem in titles and headings. Capitalize both words. See xref:security-manager[security-manager] entry for the correct usage in general text.
+==== security-manager subsystem (noun)
+*Description*: The "security-manager" subsystem is used to configure security policies used by the Java Security Manager. In general text, write in lowercase as two words separated by a hyphen. Use "Security Manager subsystem" when referring to the security-manager subsystem in titles and headings.
 
 *Use it*: yes
 
-*Incorrect forms*: Security manager, security manager, security-manager
+*Incorrect forms*:
 
-*See also*: xref:security-manager[security-manager]
+*See also*:
 
 [discrete]
 [[singleton]]
-==== singleton (noun)
-*Description*: The "singleton" subsystem is used to configure the behavior of singleton deployments. Write in lowercase. See xref:singleton-heading[Singleton] entry for the correct usage in headings.
-
-*Use it*: yes
-
-*Incorrect forms*: Singleton
-
-*See also*: xref:singleton-heading[Singleton]
-
-[discrete]
-[[singleton-heading]]
-==== Singleton (noun)
-*Description*: Use "Singleton" when referring to the xref:singleton[singleton] subsystem in titles and headings. Capitalize the word. See xref:singleton[singleton] entry for the correct usage in general text.
+==== singleton subsystem (noun)
+*Description*: The "singleton" subsystem is used to configure the behavior of singleton deployments. Write in lowercase in general text. Use "Singleton subsystem" when referring to the xref:singleton[singleton] subsystem in titles and headings.
```

Comment:
This xref can be removed (since it just refers back to the same entry

Reply:
Hi @bergerhoffer. I have removed that xref now in the latest commit.